src/main/java/com/igot/cb/cache/PromotionalContentRuleCacheMgr.java

```diff
@@ -26,7 +26,7 @@ public class PromotionalContentRuleCacheMgr {
     private final CassandraOperation cassandraOperation;
     private final CbExtServerProperties properties;
     Map<String, CachedAccessSettingRule> cacheMap = new ConcurrentHashMap<>();
-    @Value("${promotional.content.cache.ttl.milliseconds}")
+    @Value("${promotional.content.rules.cache.expiry.ms}")
     private Integer promotionalContentCacheTtlMiliSeconds;
```

Comment:
Update the variable name too -- so that if both properties are used in the same class - variable name would provide proper understanding.